torch_em/transform/augmentation.py

```diff
@@ -63,13 +68,18 @@ def apply_transform(self, input, params):
 
 # TODO implement 'require_halo', and estimate the halo from the transformations
 # so that we can load a bigger block and cut it away
-class KorniaAugmentationPipeline(torch.nn.Module):
-    interpolatable_torch_tpyes = [torch.float16, torch.float32, torch.float64]
-    interpolatable_numpy_types = [np.dtype('float32'), np.dtype('float64')]
+class AugmentationPipeline(torch.nn.Module):
```

Comment:
I think "KorniaAugmentationPipeline" makes more sense as a name, because it's not for generic Augmentations.